### PR TITLE
feat: add header

### PR DESCRIPTION
**Pull request by Hepik**
setup generic structure and add header component

image

### CODE REVIEW

File: components.json
Comment (schistostega) [inline components.json:1]: Please, consider adding a new line at the end of the file

File: src/app/globals.css [hunk 1]
```diff
@@ -2,32 +2,68 @@
...
-    text-wrap: balance;
+@layer base {
+  * {
+    @apply border-border;
   }
-}
+  body {
+    @apply bg-background text-foreground;
+  }
+}
```
Comment (schistostega) [inline src/app/globals.css]: Please, consider adding a new line at the end of the file

File: tailwind.config.ts [hunk 2]
```diff
@@ -1,20 +1,80 @@
...
       },
     },
   },
-  plugins: [],
-};
-export default config;
+  plugins: [require("tailwindcss-animate")],
+} satisfies Config
+
+export default config
```
Comment (schistostega) [inline tailwind.config.ts]: Please, consider adding a new line at the end of the file

File: components.json
Comment (Hepik) [reply components.json:1]: done

File: src/app/globals.css (hunk 1 above)
Comment (Hepik) [reply src/app/globals.css]: done

File: tailwind.config.ts (hunk 2 above)
Comment (Hepik) [reply tailwind.config.ts]: done

**Review by schistostega — APPROVED** [review]

**Pull request merged**